Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -460,6 +460,42 @@ Ref<WebPageProxy> WebPageProxy::create(PageClient& pageClient, WebProcessProxy&
     return adoptRef(*new WebPageProxy(pageClient, process, WTFMove(configuration)));
 }
 
+WebPageProxy::ProcessActivityState::ProcessActivityState(WebPageProxy& page)
+    : m_page(page)
+    , m_wasRecentlyVisibleActivity(makeUniqueRef<ProcessThrottlerTimedActivity>(8_min))
+{
+}
+void WebPageProxy::ProcessActivityState::takeVisibleActivity() { m_isVisibleActivity = m_page.process().throttler().foregroundActivity("View is visible"_s).moveToUniquePtr(); *m_wasRecentlyVisibleActivity = nullptr; }
```

Comment:
Shouldn't be on a single line in cpp files. Also, please add a blank line between functions.